common/shlibs

```diff
@@ -3862,3 +3862,4 @@ libstk-4.6.1.so libstk-4.6.1_1
 libc++utilities.so.5 cpp-utilities-5.2.0_1
 libtagparser.so.9 tagparser-9.1.2_1
 libqtutilities.so.6 qtutilities-6.0.4_1
+libkdeconnectsmshelper.so.1 kdeconnect-20.04.0_1
```

Comment:
This entry isn't needed if not other packages links against it. Also common/shlibs should always end with a new line.

Reply:
Entry removed and re-added new line.